### PR TITLE
add new page for Shared Responsibility Model

### PR DESCRIPTION
**Pull request by bethadele**
Finally adding this page to the docs site! Please take a look -- I think I made the right tweaks to get the page into the nav, but I haven't updated the docs site before so definitely check my work.

### CODE REVIEW

**Review by oredavids — DISMISSED** [review]
Markdown file and sidebar update look good.

**Comment by ebeneliason** [comment]
Glad to see this! A couple of style notes:
* Should the bold lines of text preceding each section be implemented as headers instead?
* Should we use ordered lists so that we can refer to these as e.g. section "4.b"?

**Comment by oredavids** [comment]
@bethadele reminder to look at @ebeneliason's suggestions.

File: docs/intro/overview/shared-responsibility-model.md [hunk 1]
```diff
@@ -1,49 +1,53 @@
...
+## Gruntwork is responsible for:
+
+1. Providing a tested, updated, and richly featured collection of infrastructure code for the customer to use.
+1. Maintaining a healthy Knowledge Base community where other engineers (including Grunts) post & answer questions.
+1. For Pro / Enterprise Support customers: Answering questions via email and Slack.
+1. For Reference Architecture customers:
+    1. Generating the initial Reference Architecture based on our customer’s selections of available configurations. This includes:
+        1. Our implementation of Landing Zone
+        1. A complete sample app with underlying database and caching layer
+        1. The Gruntwork Pipeline for deploying changes to infrastructure
```
Comment (zackproser) [inline docs/intro/overview/shared-responsibility-model.md:18]: NIT: `Gruntwork Pipelines` CI/CD solution for deploying changes to infrastructure

File: docs/intro/overview/shared-responsibility-model.md [hunk 2]
```diff
@@ -1,49 +1,53 @@
...
+        1. An overview of how to use the Reference Architecture
+        1. Manual steps the customer needs to take that can only be done by the customer
```
Comment (zackproser) [inline docs/intro/overview/shared-responsibility-model.md:20]: I read this section as: 

"Gruntwork is responsible for (For Reference Architecture customers) manual steps the customer needs to take that can only be done by the customer. 

Do we think this was likely intended for the other "The customer is responsible for" section? Or, if not, we should strike as it is confusing. In actuality, the customer is responsible for steps that can only be done by the customer.

File: docs/intro/overview/shared-responsibility-model.md [hunk 3]
```diff
@@ -1,49 +1,53 @@
...
+    1. Deploying the initial Reference Architecture into the customer’s brand new empty AWS accounts.
+    1. Delivering the initial Reference Architecture Infrastructure as Code to the customer.
+    1. Providing resources to the customer for deeply understanding the inner workings of the Reference Architecture.
+1. For CIS customers:
+    1. Providing IaC libraries to the CIS customer that correctly implement CIS requirements and restrictions.
+    1. For aspects of the CIS AWS Foundations Benchmark where those requirements cannot be met by modules, but require human intervention, provide instructions on manual steps the customer must take to meet the requirements.
```
Comment (zackproser) [inline docs/intro/overview/shared-responsibility-model.md:26]: ...`provide instructions...` => `providing instructions`...

File: docs/intro/overview/shared-responsibility-model.md [hunk 4]
```diff
@@ -1,49 +1,53 @@
...
+    1. For CIS Reference Architecture customers, deploying a Reference Architecture and providing access to infrastructure code that implements the CIS AWS Foundations Benchmark requirements out-of-the-box, wherever possible.
+
+## As a Gruntwork customer, you are responsible for:
+
+1. Staffing appropriately (as described in the [Prerequisites Guide](/intro/overview/reference-architecture-prerequisites-guide/)) to maintain and customize the modules and (if applicable) the Reference Architecture and to understand how the Gruntwork product works so that changes can be made to customize it to the customer’s needs.
```
Comment (zackproser) [inline docs/intro/overview/shared-responsibility-model.md:31]: "customize it to the customer's needs" => "customize it to your needs".

File: _docs-sources/intro/overview/shared-responsibility-model.md [hunk 5]
```diff
@@ -1,41 +1,45 @@
...
+    1. For aspects of the CIS AWS Foundations Benchmark where those requirements cannot be met by modules, but require human intervention, provide instructions on manual steps the customer must take to meet the requirements.
+    1. For CIS Reference Architecture customers, deploying a Reference Architecture and providing access to infrastructure code that implements the CIS AWS Foundations Benchmark requirements out-of-the-box, wherever possible.
+
+## As a Gruntwork customer, you are responsible for:
+
+1. Staffing appropriately (as described in the [Prerequisites Guide](/intro/overview/reference-architecture-prerequisites-guide/)) to maintain and customize the modules and (if applicable) the Reference Architecture and to understand how the Gruntwork product works so that changes can be made to customize it to the customer’s needs.
+    1. Raise limitations of Gruntwork modules as a feature request or a pull request.
+    1. Gruntwork does not guarantee any turn-around time on getting features built or PRs reviewed and merged. Gruntwork modules must also be applicable to a wide range of companies, and will be selective about features to be added.
```
Comment (zackproser) [inline _docs-sources/intro/overview/shared-responsibility-model.md:33]: "and will be selective" => "and we will be selective about..."

File: _docs-sources/intro/overview/shared-responsibility-model.md [hunk 6]
```diff
@@ -1,41 +1,45 @@
...
+1. Adding additional Infrastructure as Code to customize it for their company.
```
Comment (zackproser) [inline _docs-sources/intro/overview/shared-responsibility-model.md:34]: "their company" => "your company"

File: docs/intro/overview/shared-responsibility-model.md [hunk 7]
```diff
@@ -1,49 +1,53 @@
...
+    1. Raise limitations of Gruntwork modules as a feature request or a pull request.
+    1. Gruntwork does not guarantee any turn-around time on getting features built or PRs reviewed and merged. Gruntwork modules must also be applicable to a wide range of companies, and will be selective about features to be added.
+1. Adding additional Infrastructure as Code to customize it for their company.
+1. Communicating with AWS to fix account issues and limitations beyond Gruntwork’s control (quotas, account verification, et cetera).
+1. For Reference Architecture customers:
+    1. Following all provided manual steps in the Reference Architecture documents where automation is not possible.
+    1. Extending and customizing Gruntwork Pipelines beyond the basic CI/CD pipeline that Gruntwork has provided to suit their deployment requirements.
```
Comment (zackproser) [inline docs/intro/overview/shared-responsibility-model.md:38]: "their deployment requirements" => "your deployment requirements".

File: docs/intro/overview/shared-responsibility-model.md [hunk 8]
```diff
@@ -1,49 +1,53 @@
...
+    1. Designing and implementing their AWS infrastructure beyond the Reference Architecture.
```
Comment (zackproser) [inline docs/intro/overview/shared-responsibility-model.md:39]: "their" => "your"

File: docs/intro/overview/shared-responsibility-model.md [hunk 9]
```diff
@@ -1,49 +1,53 @@
...
+        1. Understanding and awareness of AWS resource costs for all infrastructure deployed into their AWS accounts ([Knowledge Base #307](https://github.com/gruntwork-io/knowledge-base/discussions/307) for Ref Arch baseline).
+    1. Once deployed, maintaining the Reference Architecture to keep it secure and up to date.
+        1. Keeping the Reference Architecture secure in accordance with their company needs.
```
Comment (zackproser) [inline docs/intro/overview/shared-responsibility-model.md:42]: "their" => "your"

**Comment by zackproser** [comment]
@ebeneliason, @oredavids and I have implemented your requested changes, and I've fixed up a bunch of voice mismatches and simplified / clarified where possible. 

I think this is ready for another look!

**Review by eak12913 — APPROVED** [review]
Stamp

**Comment by zackproser** [comment]
Thanks for review!

**Pull request merged**